opentech/apply/funds/forms.py

```diff
@@ -22,6 +22,24 @@ def __init__(self, *args, **kwargs):
         self.should_show = bool(choices)
 
 
+class ScreeningSubmissionForm(forms.ModelForm):
+    action = forms.ChoiceField(label='Update screening status')
```

Comment:
this should be: `screening_status` within the Meta fields. Then everything works as a model choice field without the setup you have in the init. Or am i missing something?

Reply:
You're not missing anything, I messed this up during my copy/pasting from another existing form.